orchestra/bots/staffbot.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, **kwargs):
         default_config = getattr(settings, 'STAFFBOT_CONFIG', {})
         default_config.update(kwargs)
         token = settings.ORCHESTRA_SLACK_STAFFBOT_TOKEN
-        super().__init__(token, **kwargs)
+        super().__init__(token, **default_config)
```

Comment:
this is unrelated to this PR, but I think it's a bug to send `kwargs` instead of `default_config` in.  @joshblum or @kkamalov any ideas?